notNeededPackages.json

```diff
@@ -2529,6 +2529,10 @@
             "libraryName": "is-lower-case",
             "asOfVersion": "1.1.2"
         },
+        "is-map": {
+            "libraryName": "is-map",
+            "asOfVersion": "2.0.4"
```

Comment:
The error is a mistake in our infrastructure, but we've never had a deprecation that matched `major.minor` before, and also happened to have more patch publishes to the types package than the source package did. We could split the greater-than version from the please-use version but the deprecation checks+message is a borderline misfeature [1]. A fix is not worth it in the general case unless it happens a few more times, and it's definitely not worth it in this case, where the types packages usage is a rounding error compared to usage of the source packages (~50k/month vs ~70M/month).

I'd recommend reverting all changes to notNeededPackages.json and doing nothing but deleting is-map/is-set files. This will signal to DT maintainers not to publish types from here anymore, but leave the already-published packages unaffected.

[1] tsc checks source package locations first, so a duplicate install of package `x` and `@types/x` ignores `@types/x`. Plus, publishing a deprecated package as `latest` can break some dependents of it.